uwblueprint/community-fridge-kw
# feat: new pickUp scheduling endpoint

## Brief description. What is this change? 

### [Create scheduling endpoints to get schedulings for pickup/not pickup](https://www.notion.so/uwblueprintexecs/Create-scheduling-endpoints-to-get-schedulings-for-pickup-not-pickup-b5e7345b689d40b586f96e0bdd2dedae?d=21d90d0727924ac58269f96c5c489bfc)



## Implementation description. How did you make this change?
* created new service called getSchedulingsByPickUp(isPickUp)
* if isPickUp is true -> returns all schedules that need pick up
* if isPickUp is false -> returns all schedules that need unloading
* updated routes, API client, services


## Steps to test
1. Create schedules that require volunteers for both pickup and unloading; as well as schedules that don't require any
2. Use postman to test 
<img width="460" alt="Screen Shot 2022-02-19 at 4 54 09 PM" src="https://user-images.githubusercontent.com/53016529/154820378-5a3a2520-7c85-4418-a6a2-f89435bf5f39.png">
<img width="448" alt="Screen Shot 2022-02-19 at 4 54 22 PM" src="https://user-images.githubusercontent.com/53016529/154820387-7ecc9db5-65ff-429e-aa65-41c523e6beaa.png">

Expected behaviour:
1. schedules that don't require volunteers should not be returned at all
2. if TRUE is passed in, return all schedules where isPickup is true
3. if FALSE is passed in, return all schedules where isPickup is false (and volunteers are required)


## Checklist
